### PR TITLE
Show all autocompletes after entry of comma in tag list

### PR DESCRIPTION
**Pull request by janmulder**
The almost 3 year old commit e1db5f21b15b2 introduced the behavior that you need to type at least 1 character for a tag to show any autocompleted data. While this went unnoticed for years, there
is always a user that notices this changed behavior.

The solution is rather simple. Do allow the 0 length string to act as seed for autocompletion instead of just returning, and doing no autocomplete at all.

As the afore mentioned commit was explicitly meant to prevent over active firing of the tag list autocompleter, this (simple) change is carefully tested on this, and no adverse effects are visible.

Fixes: #605 (wrt the 1 character issue)

Signed-off-by: Jan Mulder <jlmulder@xs4all.nl>

### Mentions:

@mturkia

### CODE REVIEW

**Comment by dirkhh** [comment]
Looks good to me, but I added @mturkia as reviewer as the Fix commit that you are unfixing :-) was his... let's wait for Miika before we merge this.

**Comment by janmulder** [comment]
Well, partially unfixing, but yes, as it is a 3 year old thing, I will wait for Miika :-)

**Comment by janmulder** [comment]
And it made me think of adding tags ... possible regression sounds serious to me, but let us simply ask for a version known to work in those cases.

**Comment by dirkhh** [comment]
I like possible-regression :-)

**Review by mturkia — APPROVED** [review]
Seems that I cannot anymore trigger the original problem. (Reverting e1db5f2 appears good to me nowadays.) Anyway, in theory, if the string is empty, one should be able to get rid of the suggestion list by deleting the comma. And thus this should not be such a pain as the original, even if some system still hits the original problem. (It was only visible on Ubuntu, and probably on specific releases of that.)

**Pull request merged**
